jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -4,7 +4,7 @@
         # TODO(fejta): consider a stable tag instead of master
         git clone https://github.com/kubernetes/test-infra -b master
         './test-infra/jenkins/bootstrap.py' \
-            --bare \
+            '{soak-repos}' \
```

Comment:
I'm pretty sure this is going to result in attempting to check out a single repo: the `k8s.io/kubernetes --repo=k8s.io/release` repo, which does not exist.

You want to check out two repos. Probably the best way to do this is to use two repos (one for the k8s repo, one for the release repo)